include/Server/include/Server.h

```diff
@@ -3,66 +3,90 @@
 #ifndef SERVER_H
 #define SERVER_H
 
-#include <boost/asio.hpp>
 #include <string>
+#include <array>
+#include <ctime>
+#include <fstream>
+#include <iostream>
+
+#include <boost/asio.hpp>
+#include <sodium.h>
 
 #include "MailMessageBuilder.h"
 #include "SocketWrapper.h"
 #include "ThreadPool.h"
 #include "MailDB/PgMailDB.h"
 #include "CommandHandler.h"
+#include "../../ErrorHandler/include/ErrorHandler.h"
+#include "../../SignalHandler/include/SignalHandler.h"
```

Comment:
This is already in the main file and is used only in it, is this include file necessary here?